### PR TITLE
Add support import Autoscaling Lifecycle Hook

### PR DESCRIPTION
**Pull request by chaspy**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request

Fixes https://github.com/terraform-providers/terraform-provider-aws/issues/9142

Release note for [CHANGELOG](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md):

```release-note
resource/aws_autoscaling_lifecycle_hook: Support resource import
```

Output from acceptance testing:

```
$ make testacc TESTARGS="-run=TestAccAWSAutoscalingLifecycleHook_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./... -v -parallel 20 -run=TestAccAWSAutoscalingLifecycleHook_basic -timeout 120m
?   	github.com/terraform-providers/terraform-provider-aws	[no test files]
=== RUN   TestAccAWSAutoscalingLifecycleHook_basic
=== PAUSE TestAccAWSAutoscalingLifecycleHook_basic
=== CONT  TestAccAWSAutoscalingLifecycleHook_basic
--- PASS: TestAccAWSAutoscalingLifecycleHook_basic (164.47s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	164.539s
```

---

Basically, same with https://github.com/terraform-providers/terraform-provider-aws/commit/46f454975737ffc83b0bca206d8b48a61b2592b4 . Thank you for the implementation and great support me @kterada0509 🙏

### CODE REVIEW

**Comment by dvishniakov** [comment]
Will this go only to v2 version of the plugin? If yes, Is it compatible with TF 0.11.x?

**Comment by chaspy** [comment]
@dvishniakov Thank you for your review 🙏 
I didn't understand `Will this go only to v2 version of the plugin?` (very shame that it is my first PR 😃 )
but I tried testing with TF 0.11.x 

```
# (build)
$ $GOPATH/bin/terraform version
Terraform v0.11.14

Your version of Terraform is out of date! The latest version
is 0.12.4. You can update by downloading from www.terraform.io/downloads.html
$ git checkout chaspy/import_aws_autoscaling_lifecycle_hook

$ make testacc TESTARGS="-run=TestAccAWSAutoscalingLifecycleHook_basic" 
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./... -v -parallel 20 -run=TestAccAWSAutoscalingLifecycleHook_basic -timeout 120m
?   	github.com/terraform-providers/terraform-provider-aws	[no test files]
=== RUN   TestAccAWSAutoscalingLifecycleHook_basic
=== PAUSE TestAccAWSAutoscalingLifecycleHook_basic
=== CONT  TestAccAWSAutoscalingLifecycleHook_basic
--- PASS: TestAccAWSAutoscalingLifecycleHook_basic (164.47s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	(cached)
```

If not enough, I'm glad if you tell me what I should do. 🙏

**Comment by chaspy** [comment]
Sorry, `make testacc` don't use local Terraform.
I'll try importing the resource with local TF with v0.11.x 👍

**Comment by chaspy** [comment]
@dvishniakov 
Tested with Terraform v11 👍 

## Target Resource
Screenshot 2019-07-16 14 29 45

## Import

```
$ $GOPATH/bin/terraform version
Terraform v0.11.14
+ provider.aws (unversioned)

Your version of Terraform is out of date! The latest version
is 0.12.4. You can update by downloading from www.terraform.io/downloads.html

$ tail -n2 autoscaling.tf 
resource "aws_autoscaling_lifecycle_hook" "chaspy_import_test_hook" {
}

$ $GOPATH/bin/terraform import aws_autoscaling_lifecycle_hook.chaspy_import_test_hook chaspy-tf-asg/import-test-hook
aws_autoscaling_lifecycle_hook.chaspy_import_test_hook: Importing from ID "chaspy-tf-asg/import-test-hook"...
aws_autoscaling_lifecycle_hook.chaspy_import_test_hook: Import complete!
  Imported aws_autoscaling_lifecycle_hook (ID: import-test-hook)
aws_autoscaling_lifecycle_hook.chaspy_import_test_hook: Refreshing state... (ID: import-test-hook)

Import successful!

The resources that were imported are shown above. These resources are now in
your Terraform state and will henceforth be managed by Terraform.
```

Please review 🙏
